demisto_sdk/__main__.py

```diff
@@ -875,8 +880,9 @@ def update_pack_releasenotes(**kwargs):
     try:
         validate_manager = ValidateManager(skip_pack_rn_validation=True, prev_ver=prev_ver)
         validate_manager.setup_git_params()
-        modified, added, old, changed_meta_files, _packs = validate_manager.get_modified_and_added_files(
-            '...', prev_ver)
+        modified, added, old, changed_meta_files = validate_manager.get_changed_files_from_git()
+        _packs = get_packs(modified).union(get_packs(old)).union(
```

Comment:
why is the `_` needed here?

Reply:
It was originaly like that, not sure why it is needed but I don't think I should change that in this PR which is large in of itself.